packages/mds-audit/api.ts

```diff
@@ -633,7 +632,7 @@ function api(app: express.Express): express.Express {
 
   app.get(pathsFor('/vehicles'), async (req, res) => {
     const { skip, take } = asPagingParams(req.query)
-    const bbox = getBoundingBox(JSON.parse(req.query.bbox))
+    const bbox = JSON.parse(req.query.bbox)
```

Comment:
hm, this feels like inadequate input validation

Reply:
Meh, this is pretty much just app-facing at the moment so I'm not super concerned -- happy to write an isBBox function though